websites/M/movie-web/metadata.json

```
@@ -0,0 +1,80 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "705490775932403795",
+		"name": "jordaar"
+	},
+	"service": "movie-web",
+	"description": {
+		"en": "The place for your favorite movies & shows"
+	},
+	"url": "movie-web.app",
+	"regExp": "(dev.)?(movie-web.app)",
```

```suggestion
	"regExp": "(dev[.])?(movie-web.app)",
```

---

The suggested regex is not working with dev.movie-web.app

---

Strange, the regex should match that domain string. I've tested it in the javascript console/regex websites.
Also, for consistency:
```suggestion
	"regExp": "(dev[.])?(movie-web[.]app)",
```